mpi-mediator/package-metadata.json

```diff
@@ -11,9 +11,23 @@
   ],
   "environmentVariables": {
     "STATEFUL_NODES": "single",
-    "CLIENT_REGISTRY_URL": "http://santedb-mpi:8080",
     "OPENHIM_MEDIATOR_URL": "https://openhim-core:8080",
-    "TRUST_SELF_SIGNED": true,
-    "REGISTER_MEDIATOR": true
+    "TRUST_SELF_SIGNED": "true",
+    "OPENHIM_USERNAME": "root@openhim.org",
+    "OPENHIM_PASSWORD": "instant101",
+    "REGISTER_MEDIATOR": "true",
```

Comment:
The MPI-mediator fails if openhim is not up, I think we should actually implement a check to see if openhim-core is up. So, if openhim is not up and REGISTER_MEDIATOR == 'true' then export REGISTER_MEDIATOR=false.

With that being said, openhim should probably become a dependency for this package.

Reply:
Yes a dependency 👍🏼